nwbinspector/utils.py

```diff
@@ -0,0 +1,26 @@
+"""Authors: Cody Baker and Ben Dichter."""
+import numpy as np
+from collections import defaultdict
+
+global default_checks
```

Comment:
rather than a global variable it may be worth creating a class ``NWBInspectorSettings`` with class variables with all the settings for the inspector. In this way you can at least collect all settings in a single place. Avoiding the use of global variable altogether would be even better, but I understand why this is needed here.